www/public/skywatcher.css

```
@@ -0,0 +1,445 @@
+    --surface-2: #22262b;
+    --ink: #e8e6df;
+    --ink-2: #b8b5ad;
+    --mute: #7a7d82;
+    --mute-2: #52565b;
+    --line: #2e3338;
+    --line-2: #23272c;
+    --acc: #d97a4f;
+    --acc-2: #5eb4c8;
+    --pos: #6bb07a;
+    --warn: #e0a64a;
+    color-scheme: dark;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)